vmdb/app/models/policy_event/purging.rb

```diff
@@ -5,7 +5,7 @@ module Purging
 
     module ClassMethods
       def purge_date
-        (purge_config(:keep_policy_events) || 6.months).to_i_with_method.ago.utc
+        (purge_config(:keep_policy_events) || 6.months).to_i_with_method.seconds.ago.utc
```

Comment:
@matthewd Merging for now. We may have to come back to this although our @mkanoor @Fryguy and my initial attempt to avoid this redundancy was not working.  Callers aren't expecting an AcitveSupport::Duration object... we can fix this if we come up with a better way.